DinnergyUITests/IngredientDeleteFeatureTest.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  IngredientDeleteFeatureTest.swift
+//  DinnergyUITests
+//
+//  Created by Jordan Roberts on 18/06/2019.
+//  Copyright © 2019 Edgy Eats. All rights reserved.
+//
+
+
+import Foundation
+import XCTest
+
+class IngredientDeleteFeatureTest: XCTestCase {
+    
+    let app = XCUIApplication()
+
+    func testDeleteIngredient(){
+        
+        addBananas(number: "4")
+        deleteFourBanana()
```

Comment:
Would appreciate it if you can explain what this line does :)

Reply:
No problem - we have created helper functions (in a file called BananaHelper.swift) to prevent repeating code when we are adding an ingredient in. Before we can test the delete ingredient feature, we have to add an ingredient to the list first for it to delete. Line 19 adds in 4kg of bananas to the list as an example. We then delete that from the list so that when we run the tests again there aren't duplicates. Does that make sense? If not, I can come and explain further :)